cips/cip-unbonding-period.md

```diff
@@ -0,0 +1,72 @@
+
+## Parameters
+
+The following parameter change is proposed for the Celestia network:
+
+| Parameter | Current value | Proposed value | Description | Changeable via Governance |
+|-----------|---------------|----------------|-------------|---------------------------|
+| `staking.UnbondingTime` | 21 days (504 hours) | 14 days + 1 hour (337 hours) | The period validators must wait after unbonding before they can withdraw their stake | No |
+| `consensus.evidence.MaxAgeDuration` | 21 days (504 hours) | 14 days + 1 hour (337 hours) | Maximum age of evidence that can be submitted for slashing | No |
+| `consensus.evidence.MaxAgeNumBlocks` | 120,960 blocks | 242,640 blocks | Maximum number of blocks for which evidence can be submitted for slashing | No |
+
+## Rationale
+
+The proposed duration of 14 days and 1 hour is carefully chosen to balance security requirements with capital efficiency:
+
+**Security Requirements:**
+
+- **IBC Light Client Compatibility**: IBC light clients currently use a 14-day trusting period. The unbonding period must exceed this to prevent validators from escaping slashing for misbehavior discovered within the trusting period
+- **Grace Period**: The additional 1 hour provides a safety buffer to ensure strict inequality (unbonding > trusting) even with timing variations
+- **Long-Range Attack Prevention**: 14+ days provides sufficient time to detect and respond to sophisticated long-range attacks or consensus failures
+
+**Capital Efficiency Benefits:**
+
+- **33% Reduction**: Reducing from 21 to ~14 days represents a 33% improvement in capital turnover time
+- **Competitive Positioning**: Aligns Celestia with industry standards (most PoS networks use 14-21 day unbonding periods)
+- **Validator Attraction**: Shorter unbonding periods make validator participation more attractive, potentially increasing decentralization
+
+## Backwards Compatibility
+
+This change introduces modifications that require careful coordination:
+
+1. **Network Upgrade Required**: The unbonding period change must be implemented through a coordinated network upgrade to ensure all validators adopt the new parameters simultaneously.
+
+2. **Existing Unbonding Delegations**: Delegations already in the unbonding process at the time of upgrade will continue under the previous 21-day period to avoid disrupting existing commitments.
+
+3. **Existing Governance Proposals to Modify Evidence Parameters**: Any existing proposals, though unlikely, must not be allowed to change the evidence params following the coordinated upgrade.
```

Comment:
Are they modifiable, and are they changed to non-modifiable after the upgrade? Looking at the table of this CIP